src/Java.Interop.Tools.Maven/Extensions/MavenNetExtensions.cs

```diff
@@ -0,0 +1,72 @@
+using System;
+using System.Diagnostics.CodeAnalysis;
+using System.Linq;
+using Java.Interop.Tools.Maven.Models;
+
+namespace Java.Interop.Tools.Maven.Extensions;
+
+static class MavenNetExtensions
+{
+	public static string GetInheritedProperty (this ResolvedDependency dependency, ResolvedProject project, Func<ResolvedDependency, string?> property)
```

Comment:
Why extension methods and not *actual* methods, when the type you're providing extension methods for is in the same assembly?  Is this so that the method isn't visible via IntelliSense by default?

Reply:
Moved to actual methods.